db/index.js

```diff
@@ -75,11 +75,12 @@ module.exports.pgp = pgp;
  *
  * @function connect
  * @param {Object} config
- * @param {function} logger
+ * @param {Object} dbLogger - logger for database log file
+ * @param {Object} logger - logger for application
  * @returns {Promise}
  * @todo Add description for the params and the return value
  */
-module.exports.connect = (config, logger) => {
+module.exports.connect = (config, dbLogger, logger) => {
```

Comment:
It seems you added new param but never used it anywhere.

Reply:
this new `dbLogger` is used for logging to database file. And the original `logger` is used [here](https://github.com/LiskHQ/lisk/pull/1919/files/f575473e78e9a180ff2a1363193064fc829f8020#diff-b13c80ec549dd76940c7f534e5d7f290R97)

Reply:
@SargeKhan It seems illogical that we log some part of db layer to one file and some part other log file. 

https://github.com/LiskHQ/lisk/blob/20d0bbab7bc5874b802344a1c31a8cdabea7dcd9/db/index.js#L95-L97

We should update this line as well to log any db error same database log file.